app/presenters/filter_presenter.rb

```diff
@@ -76,8 +76,11 @@ def organisation_options
     @organisation_options ||= begin
       additional_organisation_options +
         @organisations.map do |org|
-          name = org[:name]
-          name.concat " (#{org[:acronym]})" if org[:acronym].present?
+          name = org[:name].strip
+          if org[:acronym].present?
+            acronym = org[:acronym].strip
+          end
+          name.concat " (#{acronym})" if acronym.present? && acronym != name
```

Comment:
Could the above line be brought inside the `if`? That looks like it would allow removing the `acronym.present?` part of the conditional. The behaviour would mostly be the same, only differing for acronyms that are all whitespace.

Reply:
paired & done